easybuild/easyconfigs/g/GAMESS-US/GAMESS-US-20170420-R1-intel-2016b-sockets.eb

```diff
@@ -0,0 +1,40 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2012-2017 University of Luxembourg / LCSB, Cyprus Institute / CaSToRC,
+#                                 Ghent University / The Francis Crick Institute
+#
+# Authors::
+# * Kenneth Hoste <kenneth.hoste@ugent.be>
+# * Ward Poelmans <wpoely86@gmail.com>
+# * Oliver Stueker <oliver.stueker@ace-net.ca>
+# License::   MIT/GPL
+##
+name = 'GAMESS-US'
+version = '20170420-R1'
+versionsuffix = '-sockets'
+
+homepage = 'http://www.msg.chem.iastate.edu/gamess/index.html'
+description = """ The General Atomic and Molecular Electronic Structure System (GAMESS)
+ is a general ab initio quantum chemistry package. """
+
+toolchain = {'name': 'intel', 'version': '2016b'}
+toolchainopts = {'usempi': False}
+ddi_comm = 'sockets'
```

Comment:
this should be moved down, along with the other GAMESS-US-specific parameters (`maxcpus`, etc.)